src/points/slice.ts

```diff
@@ -0,0 +1,53 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit'
+import { ClaimHistory } from 'src/points/types'
+
+interface GetPointsHistorySucceededAction {
+  appendHistory: boolean | null
+  newPointsHistory: ClaimHistory[]
+  nextPageUrl: string | null
+}
+
+interface GetPointsHistoryStartedAction {
+  fromPage: boolean | null
```

Comment:
why boolean or null instead of just boolean? same for `appendHistory` (could these variables be named the same? maybe i'm missing something but it seems like `fromPage` becomes `appendHistory` in usage)